core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerService.scala

```diff
@@ -77,16 +82,32 @@ trait LoadBalancer {
      */
     def publish(action: ExecutableWhiskAction, msg: ActivationMessage)(implicit transid: TransactionId): Future[Future[Either[ActivationId, WhiskActivation]]]
 
+    /**
+     * Return a message indicating the health of the containers and/or container pool in general
+     * @return a Future[String] representing the heal response that will be sent to the client
+     */
+    def getHealthResponse: Future[String]
```

Comment:
This should not be a `String` but rather a `JsObject` . It needs to contain the health of each invoker specifically.